app/models/employee.rb

```diff
@@ -9,8 +9,10 @@ class Employee < ApplicationRecord
 
   has_many :given_kudos, class_name: 'Kudo', foreign_key: 'giver_id', inverse_of: 'giver', dependent: :destroy
   has_many :received_kudos, class_name: 'Kudo', foreign_key: 'reciever_id', inverse_of: 'reciever', dependent: :destroy
+  has_many :orders, dependent: :destroy
```

Comment:
It is always a moot point for me.
Do you want to delete orders for a product when it is no longer available?